test/testsuite/lantest_io_monitor.c

```diff
@@ -0,0 +1,675 @@
+/*
+ * Copyright (c) 2025, Andy Lemin (andylemin@github.com)
```

Comment:
Dunno if it's just me, but doesn't `andylemin@github.com` make look like you're a GitHub employee?

My thought was to use `@github` as a suffix without the TLD. Or maybe alternatively `@noreply.github.com` to signal that it's not a real email address? What do you think?

Reply:
Good point. When you mentioned about the Debian license expectations I figured that means it wants a valid email address. A quick google search seems to say "Copyright (C) 2025 YourGitHubUsername" is acceptable?

Reply:
Right, just your nickname/online handle is a fine substitute for the email address, I think.

So "Copyright (c) 2025, Andy Lemin (andylemin)" then?